Sources/BottomSheetPresentationController.swift

```diff
@@ -31,16 +31,23 @@ final class BottomSheetPresentationController: UIPresentationController {
         targetHeights: [CGFloat],
         startTargetIndex: Int
     ) {
-        self.targetHeights = targetHeights
-        self.startTargetIndex = startTargetIndex
+        // Add 0 height for dismiss target height
+        self.targetHeights = [0] + targetHeights
```

Comment:
Is there a way to avoid mutating target heights? We don't have to keep an array of heights in sync with array of created models (states). See my comment in: https://github.com/finn-no/BottomSheet/pull/6/files#r349659718